Build issue with cmake 3.15.4

When I try to compile rocFFT with cmake 3.5.14 I get the following error:
```
cmake -DCMAKE_CXX_COMPILER=hcc  ..
-- The CXX compiler identification is Clang 10.0.0
-- Check for working CXX compiler: /opt/rocm/hcc/bin/hcc
-- Check for working CXX compiler: /opt/rocm/hcc/bin/hcc -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- HCC compiler set; ROCm backend selected [ CXX=/opt/rocm/bin/hcc cmake ... ]
-- Building with ROCm tools
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
CMake Error at /opt/rocm/hcc/lib/cmake/hcc/hcc-config.cmake:41 (add_library):
  add_library cannot create imported target "hsa-runtime64" because another
  target with the same name already exists.
Call Stack (most recent call first):
  /home/<USERNAME>/cmake/share/cmake-3.15/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /opt/rocm/hip/lib/cmake/hip/hip-config.cmake:77 (find_dependency)
  CMakeLists.txt:151 (find_package)


-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Configuring incomplete, errors occurred!
See also "/home/<USERNAME>/repo/rocfft/d/CMakeFiles/CMakeOutput.log".
```
Using the system cmake (3.10.2), there's no issue.

--- a/lib/hcc-config.cmake.in
+++ b/lib/hcc-config.cmake.in
@@ -14,13 +14,16 @@ find_library(HSA_LIBRARY hsa-runtime64
     /opt/rocm/lib
 )
 
-add_library(hsa-runtime64 SHARED IMPORTED)
+if (NOT HSA_FOUND)
+  add_library(hsa-runtime64 SHARED IMPORTED)
 
-set_target_properties(hsa-runtime64 PROPERTIES
-  INTERFACE_INCLUDE_DIRECTORIES "${HSA_HEADER}"
-  IMPORTED_LOCATION "${HSA_LIBRARY}"
-  INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
-)
+  set_target_properties(hsa-runtime64 PROPERTIES
+    INTERFACE_INCLUDE_DIRECTORIES "${HSA_HEADER}"
+    IMPORTED_LOCATION "${HSA_LIBRARY}"
+    INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
+  )
+  set(HSA_FOUND YES)
+endif()
 
 include( "${CMAKE_CURRENT_LIST_DIR}/ImportedTargets.cmake" )
 include( "${CMAKE_CURRENT_LIST_DIR}/hcc-targets.cmake" )

